From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 24 Oct 2019 10:50:16 -0700
Subject: [PATCH] Set default options since passing them later doesn't work.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mk/config.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mk/config.mk b/mk/config.mk
index 4c53451..d26a8a7 100644
--- a/mk/config.mk
+++ b/mk/config.mk
@@ -124,7 +124,7 @@ USERLAND_INCLUDES += -I$(srcdir) -I$(builddir) -I$(top_srcdir)/include
 DESTDIR ?=
 
 # "PREFIX" part of tree, used in building other pathnames.
-PREFIX ?= /usr/local
+PREFIX ?= /usr
 
 # Compatibility with old INC_USRLOCAL which was changed to PREFIX.
 # We will overwrite PREFIX with INC_USRLOCAL until libreswan 4.2
@@ -373,7 +373,7 @@ USERLAND_CFLAGS += -DUSE_IKEv1
 endif
 
 # Enable support for DNSSEC. This requires the unbound and ldns libraries.
-USE_DNSSEC ?= true
+USE_DNSSEC ?= false
 
 # For systemd start/stop notifications and watchdog feature
 # We only enable this by default if used INITSYSTEM is systemd
